application/src/sass/_search_results.scss

```diff
@@ -109,4 +113,7 @@ main#main-content {
     font-size: 16px;
   }
 
+  .gcsc-more-maybe-branding-root {
```

Comment:
What the heck is this if we don't hide it?

Reply:
It says something like "See more results on Google?" with a link to google. 🤷‍♂️